README.md

```diff
@@ -59,8 +59,17 @@ Example of `~/.ungitrc` configuration file to change default port and enable bug
 }
 ```
 
+
+SSH Authorization
```

Comment:
The instructions from GitHub that are linked below speak of "Authentication" instead of "Authorization" which I think is the correct term in this case.

```suggestion
SSH Authentication
```